docs/bluetooth.md

```diff
@@ -0,0 +1,47 @@
+# Bluetooth
+
+This page describes the procedure to download MakeCode program to the EV3 brick 
+over Bluetooth.
+
+## ~ hint
+
+### WARNING: EXPERIMENTAL FEATURES AHEAD! 
+
+Support for Bluetooth download relies on [Web Serial](https://wicg.github.io/serial/),
+an experimental browser feature. Web Serial is a work [in progress](https://www.chromestatus.com/feature/6577673212002304); 
+it may change or be removed in future versions without notice.
+
+By enabling these experimental browser features, you could lose browser data or compromise your security 
```

Comment:
your -> security of devices connected to your computer?

Reply:
it's the verbiage of chrome
![image](https://user-images.githubusercontent.com/4175913/65773284-de6d7900-e0f0-11e9-85b5-cb3686c48e9f.png)